gulpfile.js

```diff
@@ -51,7 +51,8 @@ var lintSources = [
     srcPath,
 ].map(function (tsFolder) { return tsFolder + '/**/*.ts'; });
 lintSources = lintSources.concat([
-    '!src/typings/**'
+    '!src/typings/**',
+    '!src/test/resources/sampleReactNative022Project/**'
```

Comment:
If we're going to put commas at the end of our arrays / objects, you should add one here :)